### ISSUE TITLE
How to prevent encoding < and > like it was done in v1.4 ?

### ISSUE THREAD

**Issue by astik** [mdevils/html-entities#84]
Hi,
In v1, Html5Entities didn't encode `<>`, in v2 it is replaced by `&lt;br /&gt;`.
I have been able to tweak encode option to match v1 behavior for special character, but was not able to succeed with bracket :'(

Here is a working test case in v1:
```js
const { Html5Entities } = require('html-entities');

describe('html5 encode with v1', () => {
  it.each`
    input                        | expectation
    ${'€'}                       | ${'&#8364;'}
    ${'’'}                       | ${'&#8217;'}
    ${'é'}                       | ${'é'}
    ${'&'}                       | ${'&'}
    ${'<br />'}                  | ${'<br />'}
    ${'<a href="toto">woot</a>'} | ${'<a href="toto">woot</a>'}
  `('should transform $input into $expectation', ({ input, expectation }) => {
    const result = Html5Entities.encodeNonASCII(input);
    expect(result).toBe(expectation);
  });
});
```

Here is a failing equivalent test in v2:
```js
const { encode } = require('html-entities');

describe('html5 encode with v2', () => {
  it.each`
    input                        | expectation
    ${'€'}                       | ${'&#8364;'}                 // OK
    ${'’'}                       | ${'&#8217;'}                 // OK
    ${'é'}                       | ${'é'}                       // KO, result is &#233;
    ${'&'}                       | ${'&'}                       // KO, result is &amp;
    ${'<br />'}                  | ${'<br />'}                  // KO, result is &lt;br /&gt;
    ${'<a href="toto">woot</a>'} | ${'<a href="toto">woot</a>'} // KO, result is &lt;a href=&quot;toto&quot;&gt;woot&lt;/a&gt;
  `('should transform $input into $expectation', ({ input, expectation }) => {
    const result = encode(input, { level: 'xml', mode: 'nonAscii' });
    expect(result).toBe(expectation);
  });
});
```

I guess i can work around change for `é` but i would expect `&`, `<` or `>` to not be encoded as they are ASCII character and i'm encoding with the `nonAscii` option.
Any idea? Is this a bug or a new way to do things in this library and i would have to tackle the issue a little higher in my stack?

**Comment by mdevils** [comment]
Hello @astik.

Could you please tell me what was your use-case?

**Comment by astik** [comment]
We use this library as part of email HTML templating pipeline.
As i mention, previously, lesser than and greater than signs were not encoded and with the new version they now are.
As i'm using the nonAscii mode, i'm not expecting those signs to be encoded.
it results in those signs badly defined in our email, breaking HTML markup.

**Comment by mdevils** [comment]
@astik, please take a look at this PR: #86

Does it solve your issue?

**Comment by astik** [comment]
Look interesting indeed!
I'll take a look tomorrow with work codebase :)

**Comment by mdevils** [comment]
@astik you are welcome!

**Comment by mdevils** [comment]
@astik released in version `2.4.0`.
